packages/flutter_tools/lib/src/build_runner/resident_web_runner.dart

```diff
@@ -216,6 +217,8 @@ class ResidentWebRunner extends ResidentRunner {
       throwToolExit('Failed to connect to WebSocket.');
     } on BuildException {
       throwToolExit('Failed to build application for the Web.');
+    } on ChromeDebugException catch (err) {
+      throwToolExit(err.toString());
```

Comment:
If there's some advice that we can give along with these `throwToolExit()`s, I think that would be generally helpful even if all we can do is say "File a bug", or "Just try it again".

Reply:
Gave it another shot